src/ipelib/IPE_Forcing_Class.F90

```diff
@@ -260,7 +260,8 @@ SUBROUTINE Update_Current_Index( forcing, parameters, mpi_layer, io, deltime, rc
         line=__LINE__, file=__FILE__, rc=rc ) ) RETURN
     end if
 
-    forcing % current_index = INT( deltime / real(forcing % ifp_interval) ) + forcing % ifp_skip + 1
+    forcing % current_index = INT( (deltime - forcing % start_time / real(forcing % ifp_interval) ) &
```

Comment:
This will not build since a closing ')' is missing. It should be:
```
INT( ( deltime - forcing % start_time ) / real(forcing % ifp_interval) )
```
PRs should pass at least a build test before being issued.